src/widgets/infinite-hits/__tests__/__snapshots__/infinite-hits-test.js.snap

```diff
@@ -0,0 +1,197 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`infiniteHits() calls twice ReactDOM.render(<Hits props />, container) 1`] = `
+<InfiniteHits
+  cssClasses={
+    Object {
+      "empty": "ais-infinite-hits__empty",
+      "item": "ais-infinite-hits--item",
+      "root": "ais-infinite-hits root cx",
+      "showmore": "ais-infinite-hits--showmore",
+    }
+  }
+  hits={
+    Array [
+      Object {
+        "first": "hit",
+        "second": "hit",
```

Comment:
this seems a bit odd

Reply:
Seems odd and yet correct, no? https://github.com/algolia/instantsearch.js/blob/f87f699e3b082ae832d3fadc0bf3bed5ad8f7dff/src/widgets/infinite-hits/__tests__/infinite-hits-test.js#L33

Reply:
ohh okay, thought it would be something like that. We should rename that to have less confusion (like putting objectID and name or something in the object instead)